tools/docker-images/clp-package/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:jammy AS base
+
+WORKDIR /root
+
+COPY ./tools/docker-images/clp-package/setup-scripts ./setup-scripts
+RUN ./setup-scripts/install-prebuilt-packages.sh \
+    && rm -rf ./setup-scripts/
+
+# Remove cached files
+RUN apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+ENV CLP_HOME="/opt/clp"
+ENV PATH="${CLP_HOME}/bin:${PATH}"
+ENV PATH="${CLP_HOME}/sbin:${PATH}"
+ENV PYTHONPATH="${CLP_HOME}/lib/python3/site-packages"
```

Comment:
Below syntax reduces building layers while keeping VCS friendliness:
```
ENV KEY1=VAL1
    KEY2=VAL2
```

@coderabbitai create an issue for implementing the improvement